src/client/VZRight.tsx

```diff
@@ -1,12 +1,17 @@
-// Displays the Vite devserver iframe.
-// TODO test this out, think this through.
-// Purpose: support folks using VZCode as the editor
-// for their Vite-based projects.
-const enableIframe = false;
+import { useContext } from 'react';
+import { enableIframe } from './featureFlags';
+import { SplitPaneResizeContext } from './SplitPaneResizeContext';
 
 export const VZRight = () => {
+  const { rightPanelWidth } = useContext(
+    SplitPaneResizeContext,
+  );
+
   return (
-    <div className="right">
+    <div
+      className="right"
+      style={{ width: rightPanelWidth + 'px' }}
```

Comment:
Ok, I get the idea here, but setting the width of the right panel and not the middle panel conflicts with the way the UI is handled externally to VZCode for the VizHub environment. I would prefer to keep it such that it sets the width of the middle panel explicitly.